python/pyproject.toml

```diff
@@ -0,0 +1,80 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aac-gen-gherkin"
+version =  0.0.1
+authors = [
+    {name="AaC Dev Team", email="asdfasdaf@email.com"},
+]
+description = "An AaC plugin that generates Gherkin feature files from model behavior scenarios."
+requires-python = ">= 3.9.13"
+readme = "README.md"
+license = {text = "MIT License"}
+keywords = ["SysML", "Architecture-as-Code", "System Engineering", "MBSE"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Topic :: Scientific/Engineering"
+]
+dependencies = [
+    "aac >= 0.4.26",
+]
+
+[project.optional-dependencies]
+doc = [
+    "sphinx >= 7.3.7",
+    "sphinxcontrib-applehelp ~= 1.0.2",
+    "sphinxcontrib-devhelp ~= 1.0.2",
+    "sphinxcontrib-htmlhelp ~= 2.0.0",
+    "sphinxcontrib-jsmath ~= 1.0.1",
+    "sphinxcontrib-qthelp ~= 1.0.3",
+    "sphinxcontrib-serializinghtml ~= 1.1.9",
+    "sphinx-copybutton ~= 0.5.2",
+    "sphinx_contributors ~= 0.2.7",
+    "sphinx-autobuild ~= 2021.3.14",
+    "sphinx-simplepdf ~= 1.6.0",
+    "furo ~= 2024.5.6",
+    "docutils ~= 0.19",
+    "myst-parser ~= 2.0.0",
+    "pytz ~= 2023.3"
+]
+
+test = [
+    "build>=1.0.0",
+    "tox >= 3.24",
+    "nose2 >= 0.10.0",
+    "behave >= 1.2.6",
+    "coverage >= 6.0",
+    "flake8 >= 4.0",
+    "flake8-docstrings >= 1.6.0",
+    "flake8-fixme >= 1.1.1",
+    "flake8-eradicate >= 1.5.0",
+    "flake8-assertive >= 1.3.0",
+    "eradicate<3.0,>=2.0",
+    "httpx >= 0.23.0",
+]
+
+all = [
+    "aac >= 0.4.8",
```

Comment:
Believe you proved out that it's not necessary to include the main dependencies in the all entry anymore, so this should be able to be removed.